src/molecules/Feature.stories.mdx

```diff
@@ -0,0 +1,24 @@
UuNzc0IDI2LjM4OTkgMTUuNTMxMyAxOS4zNDY2IDIyLjI3NTJDMTIuMzAzNSAyOS4wMTkxIDEyLjA2MDkgNDAuMTk1OCAxOC44MDQ3IDQ3LjIzODlaIiBmaWxsPSJ1cmwoI3BhaW50MF9yYWRpYWwpIiBzdHJva2U9IiMxNjMxNTAiIHN0cm9rZS13aWR0aD0iMi42MDc2IiBzdHJva2UtbWl0ZXJsaW1pdD0iMTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPgo8cGF0aCBkPSJNMjUuODY5OSAyNS44NjE4QzMzLjU4MjggMjEuMDA1NCA0My40Mjc4IDI3LjAzMjMgNDIuODAxMiAzNi4wNjE1TDMyLjEyNjcgMzUuMTgzOUwzOC45MDI4IDQzLjcyOEMzMy44NTgxIDQ3Ljk5NjQgMjYuMDU5MiA0Ni44OTQgMjIuMzgwMyA0MS4xNDQ1QzE5LjExOTMgMzUuOTc4MSAyMC43MDM1IDI5LjEyMjcgMjUuODY5OSAyNS44NjE4WiIgZmlsbD0iI0ZGQzgzNSIgc3Ryb2tlPSIjRkZBRTI1IiBzdHJva2Utd2lkdGg9IjIuNjA3NiIgc3Ryb2tlLW1pdGVybGltaXQ9IjEwIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiLz4KPHBhdGggZD0iTTQxLjM4OTEgNDkuMDAySDUuNjAxMDdWNjUuNTI2NEgxMy43MDEzQzE0LjMwNzEgNjUuNTI2NCAxNC45MDk4IDY1LjYxMDcgMTUuNDkyMiA2NS43NzcxTDMzLjIzNTQgNzAuODQ2N0MzNC43ODY1IDcxLjI4OTggMzYuNDQ3IDcxLjE0MjEgMzcuODk1NSA3MC40MzE5TDY5Ljk5OTYgNTQuNjk0N0M3Mi4xNzcxIDUzLjYyNzQgNzMuMDY3NiA1MC45OTAyIDcxLjk4MzIgNDguODIxM0M3MC45NzE1IDQ2Ljc5NzkgNjguNTgyIDQ1Ljg4MjMgNjYuNDc3MyA0Ni43MTE0TDQ4LjgyOTIgNTMuNjYzN0M0Ny4wMTcgNTQuMzc3NiA0NC45OTA0IDUzLjMzNDcgNDQuNTE4MSA1MS40NDUxQzQ0LjE1OTEgNTAuMDA5MyA0Mi44NjkgNDkuMDAyIDQxLjM4OTEgNDkuMDAyWiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZD0iTTguNjA3OTEgNDkuMDAxN0g0MC42MTI2QzQzLjI2MzEgNDkuMDAxNyA0NS40MTE5IDUxLjE1MDQgNDUuNDExOSA1My44MDFDNDUuNDExOSA1Ni4yMTE2IDQzLjYyMzggNTguMjQ4MSA0MS4yMzM0IDU4LjU1OTlMMjguMTM2NiA2MC4yNjgyIiBzdHJva2U9IiMxNjMwNTAiIHN0cm9rZS13aWR0aD0iMi42MDc2IiBzdHJva2UtbWl0ZXJsaW1pdD0iMTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPgo8cGF0aCBkPSJNOC42MDc5MSA2NS41MjY0SDE0LjYxNjdMMzIuMDI1NiA3MS4xMjJDMzQuMzQzNiA3MS44NjcxIDM2Ljg2MzIgNzEuNjU1IDM5LjAyNDEgNzAuNTMzMUw3MC41MjEzIDU0LjE3ODhDNzIuNzU0MiA1My4wMTkzIDczLjQ4NTcgNTAuMTc3OCA3Mi4wOSA0OC4wODQzQzcwLjk4NjIgNDYuNDI4NCA2OC44NzY3IDQ1Ljc2NzMgNjcuMDI1MSA0Ni40OTY2TDQ1LjQxMTkgNTUuMDExIiBzdHJva2U9IiMxNjMwNTAiIHN0cm9rZS13aWR0aD0iMi42MDc2IiBzdHJva2UtbWl0ZXJsaW1pdD0iMTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPgo8cGF0aCBkPSJNMTQuNzI4NCA0Ni41NDk2SDEuNjkwNDNWNzAuMDE4SDE0LjcyODRWNDYuNTQ5NloiIGZpbGw9IiMwNUQ1RkMiLz4KPHBhdGggZD0iTTE0LjcyODQgNjMuNjE3N0gxLjY5MDQzVjcwLjAxODFIMTQuNzI4NFY2My42MTc3WiIgZmlsbD0iIzFFQjhFNyIvPgo8cGF0aCBkPSJNMTQuNzI4NCA0Ni41NDk2SDEuNjkwNDNWNzAuMDE4SDE0LjcyODRWNDYuNTQ5NloiIHN0cm9rZT0iIzE2MzA1MCIgc3Ryb2tlLXdpZHRoPSIyLjYwNzYiIHN0cm9rZS1taXRlcmxpbWl0PSIxMCIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIi8+CjxkZWZzPgo8cmFkaWFsR3JhZGllbnQgaWQ9InBhaW50MF9yYWRpYWwiIGN4PSIwIiBjeT0iMCIgcj0iMSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiIGdyYWRpZW50VHJhbnNmb3JtPSJ0cmFuc2xhdGUoMzEuMDA5NSAyMS4xMTk3KSByb3RhdGUoNzkuMjM2KSBzY2FsZSgxNC40MDI2KSI+CjxzdG9wIG9mZnNldD0iMC4wNjQ4NjQ5IiBzdG9wLWNvbG9yPSIjRkZFQTgwIi8+CjxzdG9wIG9mZnNldD0iMC4xMDg3IiBzdG9wLWNvbG9yPSIjRkZFNzdEIi8+CjxzdG9wIG9mZnNldD0iMC40Njg5IiBzdG9wLWNvbG9yPSIjRkZENzZDIi8+CjxzdG9wIG9mZnNldD0iMC43MzUiIHN0b3AtY29sb3I9IiNGRkQxNjYiLz4KPC9yYWRpYWxHcmFkaWVudD4KPC9kZWZzPgo8L3N2Zz4K',
```

Comment:
Do we really want this 😅

Reply:
It's just for the story. Shouldn't really matter, right?